charactersheet/charactersheet/services/common/sync/card_service.js

```diff
@@ -107,6 +107,11 @@ function _pCardService(configuration) {
 
         if (pCardInParty) {
```

Comment:
This logic should be in the Image Service and should be fired on either `pCard.changed` or `players.changed`. Also there's no need to do the null check since dispatching `list[0]` will do the same thing functionally.